### PR TITLE
data: manishitMani

### PR DESCRIPTION
**Pull request by manishitMani**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5450"></a>

### CODE REVIEW

File: data/manishitMani.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Manishit Mani",
+  "displayStatsPublic": "true",
```
Comment (imbeshat) [inline data/manishitMani.json:3]:
```suggestion
```

File: data/manishitMani.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "I am a devloper",
+  "avatar": "http://github.com/manishitMani.png",
```
Comment (imbeshat) [inline data/manishitMani.json:6]:
```suggestion
```

File: data/manishitMani.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/manishit080",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on Github",
+      "url": "https://github.com/manishitMani",
+      "icon": "FaGithubAlt"
+    }
+
```
Comment (imbeshat) [inline data/manishitMani.json:18]:
```suggestion
    ]
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/manishitMani.json (hunk 1 above)
Comment (imbeshat) [reply data/manishitMani.json:3]: not required

File: data/manishitMani.json (hunk 3 above)
Comment (kumarsonsoff3) [reply data/manishitMani.json:16]: Hey @manishitMani, Still there is missing comma leading json check fail. Please add a comma as suggested above. 🛠

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**